recipe_scrapers/sunset.py

```diff
@@ -1,7 +1,41 @@
 from ._abstract import AbstractScraper
+from ._grouping_utils import IngredientGroup
 
 
 class Sunset(AbstractScraper):
     @classmethod
     def host(cls):
         return "sunset.com"
+
+    def ingredient_groups(self):
+        groups = []
+        current_group = []
+        current_purpose = None
+        seen_ingredients = set()
```

Comment:
Let's remove the `seen_ingredients` set now that it is (hopefully!) not required any more?